src/modules/land_detector/MulticopterLandDetector.cpp

```diff
@@ -172,7 +172,8 @@ bool MulticopterLandDetector::_get_landed_state()
 	// Check if user commands throttle and if so, report not landed based on
 	// the user intent to take off (even if the system might physically still have
 	// ground contact at this point).
-	if (_manual.timestamp > 0 && _manual.z > 0.15f && _ctrl_mode.flag_control_manual_enabled) {
```

Comment:
This was causing the landing detector to detect in_air in position control when the drone was on the ground idling and the user was raising his throttle stick.